docs/demos/mpi.ipynb

```diff
@@ -0,0 +1,307 @@
+{
```

Comment:
<p>Again, just print the total time, instead of a single iteration time</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/steven-murray/powerbox/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>